### PR TITLE
file: readme.md file added [#20]

### PR DESCRIPTION
**Pull request by anjaliahlawat**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+The script runs a script to test the following specifications:
+- Amazon login page  
+- Whether user has successfully logged into amazon account
+
+The project uses Node library [Puppeteer](https://devdocs.io/puppeteer/) and runs test cased on [Gauge](https://docs.gauge.org/overview.html?os=linux&language=javascript&ide=vscode).
+
+## Prerequisites 
+
+Before you begin, please ensure you have the following setup in your system as per environment of your choice
+- Node.js
```
Comment (h2grover) [inline README.md]: please specify the node version which is required and also link the nvm installation guide. It is always better to use nvm to install the different node versions.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+- Package manager - npm or yarn
```
Comment (h2grover) [inline README.md]: link the `yarn` installation page. `npm` is not needed so you can remove it.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+## Installation Guide
+
+To run this project, clone the repository, open the directory and run the following commands:
+
+```
+$ yarn add
```
Comment (h2grover) [inline README.md]: shouldn't it be `yarn` or `yarn install` ?

File: README.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+$ yarn test
```
Comment (h2grover) [inline README.md:27]: this will not come under installation step

File: README.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+```
+
+
```
Comment (h2grover) [inline README.md:39]: remove extra lines.

**Review by h2grover — CHANGES_REQUESTED** [review]
Its better to add commands from gauge like, how to run in parallel , tags specific.

File: src/lib/PageWrapper.js [hunk 6]
```diff
@@ -1,24 +1,28 @@
-const { v4: uniqueId } = require('uuid');
+const { v4: uniqueId } = require("uuid");
```
Comment (h2grover) [inline src/lib/PageWrapper.js:1]: @anjaliahlawat how come these changes are coming here?
it should already be in `main` branch

Comment (h2grover) [reply src/lib/PageWrapper.js:1]: image


Why you have force pushed your changes?

Comment (anjaliahlawat) [reply src/lib/PageWrapper.js:1]: I did pull the changes. But some linting errors were still shown when I tried to commit so I fixed them here again.

Comment (anjaliahlawat) [reply src/lib/PageWrapper.js:1]: > image
> 
> Why you have force pushed your changes?

I committed incomplete changes. So I forced it back to previous commit and completed the changes and then pushed it here.

Comment (h2grover) [reply src/lib/PageWrapper.js:1]: please do not force push any changes moving forward.

**Review by h2grover — APPROVED** [review]
@anjaliahlawat 
I am merging this PR for now. But moving forward any refactoring or enhancements that are not PR related should be done in a separate PR first. in this case, it was linting changes. 

Correct process:
1. Create separate PR for the lint fixes
1. I will merge that PR after review
1. You will then merge `main` branch to this PR 
1. Then this PR will get merged.

**Pull request merged**